legacy/crypto/walletManager.js

```diff
@@ -128,6 +128,27 @@ class WalletManager {
           checksum = w.checksum
         }
         const isHW = data != null && data.isHW != null ? data.isHW : false
+
+        // migration - missing checksum
+        const metadataUpdate: WalletMeta = {...w}
```

Comment:
can this be pulled out and given a name?
would it make sense to add a test for these 2 migrations?

Reply:
🤔